File: capbox/lib/features/admin/data/dtos/clave_gimnasio_dto.dart
/// DTO para la clave del gimnasio retornado por GET/PATCH /v1/profile/gym/key
class ClaveGimnasioDto {
  final String claveGym;

  const ClaveGimnasioDto({required this.claveGym});

  factory ClaveGimnasioDto.fromJson(Map<String, dynamic> json) {
    return ClaveGimnasioDto(claveGym: json['claveGym'] as String);
  }

  Map<String, dynamic> toJson() {
    return {'claveGym': claveGym};
  }

  @override
  String toString() {
    return 'ClaveGimnasioDto{claveGym: $claveGym}';
  }
}

File: ms-identidad/src/infraestructura/dtos/clave-gimnasio.dto.ts
/**
 * DTO para la respuesta del endpoint que obtiene la clave de registro de un gimnasio.
 */
export class ClaveGimnasioDto {
  /**
   * La clave de registro única del gimnasio.
   * @example "GYM-CAMPEONES-2025"
   */
  claveGym: string;
}
